narwhal/config/src/committee.rs

```diff
@@ -105,6 +106,9 @@ pub struct Committee {
     authorities_by_id: BTreeMap<AuthorityIdentifier, Authority>,
     /// The epoch number of this committee
     epoch: Epoch,
+    // Protocol version for this epoch.
+    #[serde(skip)]
+    protocol_version: ProtocolVersion,
```

Comment:
I would propose to either set this to some default (example MAX) instead of relying to default.

Reply:
We should pass in ProtocolConfig, not ProtocolVersion. Directly examining ProtocolVersion in code is not the pattern we generally use